content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_two_steps.rst

```diff
@@ -4,185 +4,229 @@ Process receipts and deliveries in two steps
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-new-sales-order.png
    :align: center
-   :alt: Delivery smart button on sales order form.
+   :alt: Filled out new sales order form.
 
-.. tip::
-   Sales order receipts can also be found in the :menuselection:`Inventory` application. In the
-   :guilabel:`Overview` dashboard, click the :guilabel:`# To Process` smart button in the
-   :guilabel:`Pick` Kanban card.
+Under the :guilabel:`Product` tab, click :guilabel:`Add a product`, and select a product to add to
+the sales order quotation.
+
+Once ready, click :guilabel:`Confirm`. This moves the quotation to the :guilabel:`Sales Order`
+stage.
+
+Once the |SO| is confirmed, a :guilabel:`Delivery` smart button appears at the top of the form.
+Clicking the smart button opens the warehouse delivery (WH/OUT) form.
+
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-delivery-button.png
+   :align: center
+   :alt: Delivery smart button on validated sales order form.
 
-Process the picking
--------------------
+Process picking
+---------------
 
 Once the sales order is confirmed, a picking order is generated and ready to process.
 
-Navigate to the :menuselection:`Inventory app`, and locate the :guilabel:`Pick` task card on the
-:guilabel:`Inventory Overview` dashboard. This reveals a list of all pickings to process.
+To complete the picking, navigate to the :guilabel:`Inventory app`, and locate the :guilabel:`Pick`
+task card on the :guilabel:`Inventory Overview` dashboard.
+
+Click the :guilabel:`# To Process` button on the task card. This reveals a list of all pickings to
+process.
 
 Click on the picking (WH/PICK) operation associated with the sales order to reveal the picking
 order.
 
+.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
+   :align: center
+   :alt: Picking order form for products included in sales order.
+
 Click :guilabel:`Check Availability` on the picking order form. If the product is in stock, Odoo
 automatically reserves the product. Quantities can also be manually set by changing the value in the
 :guilabel:`Quantity` column to match the value in the :guilabel:`Demand` column.
 
-Once ready, click :guilabel:`Validate` to complete the picking and move the product from
+Once ready, click :guilabel:`Validate` to complete the picking, and move the product from
 :guilabel:`WH/Stock` to :guilabel:`WH/Output.`
 
-.. image:: receipts_delivery_two_steps/receipts-delivery-two-steps-picking-form.png
-   :align: center
-   :alt: Picking operation form moving product to WH/Output.
-
-Process the delivery
---------------------
-
-Once the picking has been processed, the delivery order is generated, and can be processed.
+Process delivery
+----------------
 
-Navigate back to the original sales order by going to :menuselection:`Sales app`, and selecting the
-associated sales order.
+Once the picking is validated, a delivery order is created and ready to process.
```

Comment:
```suggestion
Once the picking is validated, a delivery order is created and ready to process.
```

This was created for me already when confirming the sales order

Reply:
In 17.2 it's not created until after validation of the previous order

Reply:
This is another instance where I think you'll need to double-check things to ensure accuracy, because, even when I check things on a 17.2 runbot, Tim and I are getting the same results.

Reply:
What is generated upon confirmation of an SO is not a delivery order, it's the delivery smart button - which if clicked contains the picking order. Only when the picking order is validated does an actual delivery order generate - which is also contained in the delivery smart button. I'll reword this section to clarify this, but it is functionally correct as is